source/settings/plugin/autocreate-plugin.rst

```diff
@@ -4,6 +4,8 @@
 autocreate plugin
 ===================
 
+.. versionremoved:: 2.3.14 Use autocreate namespace mailbox setting instead.
+
 With v2.1+ you don't need this plugin. Use `mailbox { auto } <https://wiki.dovecot.org/MailboxSettings>`_ setting instead.
```

Comment:
These two lines could be combined in some way. Also would be really nice if the wiki text was moved to doc :)

Reply:
They could both link to https://doc.dovecot.org/configuration_manual/namespace/ - and even better if there was #auto anchor.

Reply:
or maybe https://doc.dovecot.org/configuration_manual/namespace/#mailbox-settings

Reply:
@jeffpc  ping

Reply:
Like that?